src/views/salasana.hbs

```diff
@@ -0,0 +1,99 @@
+<!-- -------------------------------------------------------------------------- -->
+<!-- Salasana body -->
+<!-- -------------------------------------------------------------------------- -->
+
+<div id="bodywrapper" class="viewport-full flex-column">
+
+  <!-- ------------------------------------------------------------------------ -->
+  <!-- Navigation bar -->
+  {{> sharedPartials/navbar }}
+
+  <!-- ------------------------------------------------------------------------ -->
+  <!-- Progress bar -->
+  <div id="progressbar" class="progress-bar-inactive"></div>
+
+  <!-- ------------------------------------------------------------------------ -->
+  <!-- Application -->
+  <div id="root">
+
+    <!-- ---------------------------------------------------------------------- -->
+    <!-- vaihto tab -->
+    <div id="vaihto" class="tab-wrap">
+      <form id="change" class="form-container">
+        <div class="form-content">
+          <div class="form-content-section direction-row">
+            <div class="filler"></div>
+            <div class="column-wrap">
+              <div id="changeFormPassword" class="form-field-container full-width">
+                <div class="label-wrapper">
+                  <label for="change-password">
```

Comment:
A label should be linked with one form control element, so the value for attribute _for_ should be the input element's _id_ `<label for="change-password-org">`

Reply:
fixed